source/reference/minio-mc/mc-idp-ldap-accesskey-create-with-login.rst

```diff
@@ -0,0 +1,200 @@
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+         The following example prompts the user to provide their AD/LDAP credentials.
+         It then generates a new access key pair using the policy or policies associated with that AD/LDAP user.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc idp ldap accesskey create-with-login https://minio.example.net/
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] idp ldap accesskey create-with-login        \
+                                          URL                         \
+                                          [--access-key <value>]      \
+                                          [--secret-key <value>]      \
+                                          [--policy <value>]          \
+                                          [--name <value>]            \
+                                          [--description <value>]     \
+                                          [--expiry <value>]          \
+                                          [--expiry-duration <value>]
+
+      - Replace ``URL`` with the :abbr:`FQDN (Fully Qualified Domain Name)` of a MinIO deployment configured for AD/LDAP integration.
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: URL
+   :required:
+
+   the :abbr:`FQDN (Fully Qualified Domain Name)` of a MinIO deployment configured for AD/LDAP integration.
+
+   For example:
+
+   .. code-block:: none
+
+         mc idp ldap accesskey create-with-login https://minio.example.net
+
+.. mc-cmd:: --access-key
+   :optional:
+
+   The access key to use once successfully authenticated.
+   Omit to let MinIO randomly generate a value.
+   
+   The access key cannot contain the characters ``=`` (equal sign) or ``,`` (comma).
+
+   Requires :mc-cmd:`~mc idp ldap accesskey create-with-login --secret-key`
+
+.. mc-cmd:: --secret-key
+   :optional:
+
+   A secret key to use once successfully authenticated.
+   Omit to let MinIO randomly generate a value.
+
+   Requires :mc-cmd:`~mc idp ldap accesskey create-with-login --access-key`
+
+.. mc-cmd:: --policy
+   :optional:
+
+   File path to the JSON-formatted :ref:`policy <minio-policy>` to use for the account.
+   This policy can only further limit those privileges associated with the authenticated AD/LDAP user.
+
+   Omit to use the AD/LDAP user policies.
+
+.. mc-cmd:: --name
+   :optional:
+
+   A human-readable name to use for the created access key.
+
+.. mc-cmd:: --description
+   :optional:
+
+   Add a description for the service account.
+   For example, you might specify the reason the access key exists.
+
+.. mc-cmd:: --expiry-duration
+   :optional:
+
+   Length of time the access key pair should remain valid for use in ``#d#h#s`` format.
```

Comment:
A question about the behavior: you can specify some combination of days, hours, or seconds, but not _minutes_?

Reply:
I am not sure - will have to check and maybe patch that later